src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area,Drawable {
```

Comment:
The `Area` and `Drawable` interfaces should not be implemented together if their methods are conceptually different. Consider separating `getArea` and `draw` into different structures if they don't serve the same purpose.